tests/test_code_format.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+from unittest import TestCase
+
+from pycodestyle import StyleGuide
+
+
+class TestCodeFormat(TestCase):
+    """ This test class should ensure all earthdata-varinfo Python code adheres
+        to standard Python code styling.
+
+        Ignored errors and warnings:
+
+        * E501: Line length, which defaults to 80 characters. This is a
+                preferred feature of the code, but not always easily achieved.
+        * W503: Break before binary operator. Have to ignore one of W503 or
```

Comment:
Not a nit or suggestion but what's this binary operator business about haha?

Reply:
[Here's the biblical reference](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).

An example if this is whether you should have:

```
if (
    my_value == 1.2345
    or my_other_value == 2.3456
):
    ...
```

Or:

```
if (
    my_value == 1.2345 or
    my_other_value == 2.3456
):
    ...
```

Ultimately the answer is that either are okay, so long as the code is consistently one or the other, with PEP008 stating a mild preference for the first option. `pycodestyle` is automatically configured to warn against _both_ options, which is unhelpful, so the test ignores the checking of the one that is considered slightly preferred.

Reply:
Yeah - either is okay, but we should choose one and it throughout the project. Then I turned off the warning for the one we went with. So that second example will still trigger a warning, but the first one won't.